spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -609,6 +626,30 @@ def upload_trailer!(trailer_path, sort_order, language, device, timestamp = "05.
         setup_trailers
       end
 
+      # Uploads, app review attachments
+      #
+      # while submitting for review, ITC allow developers to attach file.
+      #
+      # File types allowed by Apple are: pdf, doc, docx, rtf, pages, xls, xlsx, numbers
+      # zip, rar, plist, crash, jpg, png, mp4 or avi.
```

Comment:
Is there more missing than just a mime type for fastlane to support more of these formats? At least the flat-file formats might be nice if they’re easy to add. (Note: .numbers can be a folder or a file.)

Reply:
done

Reply:
Sweet! Thank you! Could you update the comment about supported types in app_version.rb? Thanks!

I hope to try this out Tuesday or Wednesday this week.

Reply:
Nevermind. I see you updated the comment here. Thank you!